families: 'tag36h11'
nthreads: 1
quad_decimate: 2.0
quad_sigma: 0.0
refine_edges: 1
decode_sharpening: 0.25
tag_size: 0.072
publish_rate: 10
publish_tf: true # default: false